Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -218,7 +218,7 @@ class BrowserViewController: UIViewController {
         })
         pasteAction = AccessibleAction(name: NSLocalizedString("Paste", comment: "Paste the URL into the location bar"), handler: { () -> Bool in
             if let pasteboardContents = UIPasteboard.generalPasteboard().string {
-                self.urlBar.updateURLBarText(pasteboardContents)
+                self.urlBar.enterOverlayMode(pasteboardContents)
```

Comment:
OK, but it's confusing to me for a couple reasons. For one, text fields already have an `editing` property that's basically whether they're focused or not, so it's easy to confuse the two. Second, we're still in "editing mode" even when we're interacting with the panels, and we're not actually editing anything...

That said, no strong preference here, so I can change it. Or maybe something else is better? `ModalMode`? `HomeMode`?